odoo/service/server.py

```diff
@@ -609,6 +609,7 @@ def start(self, stop=False):
             signal.signal(signal.SIGXCPU, self.signal_handler)
             signal.signal(signal.SIGQUIT, dumpstacks)
             signal.signal(signal.SIGUSR1, log_ormcache_stats)
+            signal.signal(signal.SIGUSR2, log_ormcache_stats_with_size)
```

Comment:
Does the performance different justify the use of SIGUSR2?
Since you do it in a separate thread anyways, I would not bind a new signal to that same feature.